packages/react/src/themes/teams/siteVariables.ts

```diff
@@ -11,35 +11,21 @@ export const htmlFontSize = '10px' // what 1rem represents
 //
 export { colors, contextualColors, emphasisColors, naturalColors, colorScheme } from './colors'
 
-export const black = colors.grey[900]
-export const gray02 = '#484644'
-export const gray03 = '#605E5C'
-export const gray04 = '#979593'
-export const gray06 = '#C8C6C4'
-export const gray08 = '#E1DFDD'
-export const gray09 = '#EDEBE9'
-export const gray10 = '#F3F2F1'
-export const gray14 = '#FAF9F8'
+export const gray02 = '#484644' // no mapping color
+export const gray03 = '#605E5C' // no mapping color
+export const gray04 = '#979593' // no mapping color
+export const gray06 = '#C8C6C4' // no mapping color
+export const gray08 = '#E1DFDD' // no mapping color
+export const gray09 = '#EDEBE9' // no mapping color
+export const gray10 = '#F3F2F1' // no mapping color
+export const gray14 = '#FAF9F8' // no mapping color
```

Comment:
Have a mixed feeling about it. If we are trying to rid of colors on `siteVariables` directly, should we keep these?
May be add an evil prefix to them? `__unsafe__gray14`? 😈

Reply:
This step is just a clean up, so that we can have clear picture of what colors are missing in the color palette. Then the design team can safely update the color palette, following the steps introduced in this PR. I would not want to introduce breaking changes now for these colors, as most likely they are going to be removed soon.